ci-operator/step-registry/ipi/conf/aws/user-min-permissions/ipi-conf-aws-user-min-permissions-commands.sh

```diff
@@ -295,6 +295,13 @@ EOF
 			echo "elasticloadbalancing:SetSecurityGroups" >>"${PERMISION_LIST}"
 			echo "s3:PutBucketPolicy" >>"${PERMISION_LIST}"
 		fi
+
+		# additional permisions for 4.17+
+		# https://issues.redhat.com/browse/OCPBUGS-44848
+		if ((ocp_minor_version >= 17 && ocp_major_version == 4)); then
```

Comment:
Similar to https://github.com/openshift/release/pull/61374#discussion_r1946121648, I guess this permission is only needed for byo-iam-profile case, right? If right, maybe we can  read install-config and assign proper permissions instead of checking version.

Reply:
And I saw `OCPBUGS-44848` is already fixed, do we still need to append the permission manually ?

Reply:
Yes. Installer will have a pre-check after `OCPBUGS-44848` fix applied. The permission is mandatory now if byo IAM profile is configured.

Reply:
Ah, for older version, we are creating IAM mini-perm users by manually creating permission list, installer only run some pre-check.